packages/lib/src/components/internal/Await/Await.tsx

```diff
@@ -106,15 +106,7 @@ function Await(props: AwaitComponentProps) {
     };
 
     useEffect(() => {
-        const { shouldRedirectOnMobile, url } = props;
-        const isMobile: boolean = window.matchMedia('(max-width: 768px)').matches && /Android|iPhone|iPod/.test(navigator.userAgent);
-
-        if (shouldRedirectOnMobile && url && isMobile) {
-            redirectToApp(url, checkStatus);
```

Comment:
In the [redirectToApp](https://github.com/Adyen/adyen-web/blob/4d55ec1349a83fb6694238c2eebb2a16f47a7add/packages/lib/src/components/internal/Await/Await.tsx#L104) function, the `setTimeout` can be removed (same as you did in the QRLoader.tsx)